cli/coverage_cmd.go

```diff
@@ -329,7 +345,7 @@ func stripComments(data []byte) []byte {
 	tok := s.Scan()
 	for tok != scanner.EOF {
 		pos := s.Pos()
-		if tok == scanner.Comment {
+		if tok != scanner.Ident {
```

Comment:
Is the Go scanner accurate enough for other languages? I think it is, but perhaps it's worth adding a comment for a future reader of the code (some languages' comment syntax, e.g., Python, is a bit weird).